test/subscription.js

```diff
@@ -840,3 +840,101 @@ test('subscription server sends correct error if execution throws', t => {
     })
   })
 })
+
+test('subscription server exposes pubsub', t => {
+  // t.plan(2)
```

Comment:
why is this commented out?

Reply:
This is not required, the test is working without t.plan